### PR TITLE
07 web UI testing adding demo file

### PR DESCRIPTION
**Pull request by SSN-SPB**

### CODE REVIEW

**Review by ericrommel — APPROVED** [review]

File: topics/07_web_ui_testing/example_of_test_suites_and_po/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Introduction
+This folder contains files with PO and test suites to demo different implementation of Web UI automation tests
+
+
+## Files
+
+1)
+selenium_test_demo.py:
```
Comment (ericrommel) [inline topics/07_web_ui_testing/example_of_test_suites_and_po/README.md:8]: Put the file names in the same line as the number items and add a backtick to better style. Example:
```
1. `selenium_test_demo.py`: It is a simple example of selenium webdriver test. CLI command:
   \```bash
   $ python .\selenium_test_demo.py
   \```
```

The above slashes were only to demonstrate. Do this change for all of the next ones.

**Review by ericrommel — CHANGES_REQUESTED** [review]
See my comment.

File: topics/07_web_ui_testing/example_of_test_suites_and_po/README.md (hunk 1 above)
Comment (SSN-SPB) [reply topics/07_web_ui_testing/example_of_test_suites_and_po/README.md]: Done.
Review it, please.

**Review by ericrommel — APPROVED** [review]
LGTM

**Comment by SSN-SPB** [comment]
Ready to merge

**Pull request merged**